core/tests/protractor/editorAndPlayer.js

```diff
@@ -18,15 +18,17 @@
 
 var general = require('../protractor_utils/general.js');
 var forms = require('../protractor_utils/forms.js');
-var users = require('../protractor_utils/users.js');
+var UsersPage = require('../protractor_utils/UsersPage.js');
 var workflow = require('../protractor_utils/workflow.js');
 var editor = require('../protractor_utils/editor.js');
 var player = require('../protractor_utils/player.js');
 
+var usersPage = new UsersPage.UsersPage()
+
 describe('Full exploration editor', function() {
   it('should navigate multiple states correctly, with parameters', function() {
-    users.createUser('user4@editorAndPlayer.com', 'user4EditorAndPlayer');
-    users.login('user4@editorAndPlayer.com');
+    usersPage.createUser('user4@editorAndPlayer.com', 'user4EditorAndPlayer');
```

Comment:
I would love to see these values (`'user*@editorAndPlayer.com`, `user*EdtorAndPlayer`) hidden behind an array of constants. Maybe something like:

```javascript
var USERS = [
    {name: 'user4EditorAndPlayer', address: 'user4@editorAndPlayer.com'},
    // ...
    {name: 'user8EditorAndPlayer', address: 'user8@editorAndPlayer.com'},
]
```

Then, the test code can reference these values by index instead:

`usersPage.createUser(USERS[0].name, USERS[0].address);`

Could you please try this out?

Reply:
yeah sure!!